medusa/service/grpc/restore.py

```diff
@@ -48,36 +49,59 @@ def configure_console_logging(config):
             logging.getLogger(logger_name).setLevel(logging.WARN)
 
 
-if len(sys.argv) > 2:
-    config_file_path = sys.argv[2]
-else:
-    config_file_path = "/etc/medusa/medusa.ini"
-
-config = create_config(config_file_path)
-configure_console_logging(config.logging)
-
-backup_name = os.environ["BACKUP_NAME"]
-tmp_dir = Path("/tmp")
-in_place = True
-keep_auth = False
-seeds = None
-verify = False
-keyspaces = {}
-tables = {}
-use_sstableloader = False
-
-cluster_backups = medusa.listing.get_backups(config, False)
-backup_found = False
-# Checking if the backup exists for the node we're restoring.
-# Skipping restore if it doesn't exist.
-for cluster_backup in cluster_backups:
-    if cluster_backup.name == backup_name:
-        backup_found = True
-        logging.info("Starting restore of backup {}".format(backup_name))
-        medusa.restore_node.restore_node(config, tmp_dir, backup_name, in_place, keep_auth,
-                                         seeds, verify, keyspaces, tables, use_sstableloader)
-        logging.info("Finished restore of backup {}".format(backup_name))
-        break
-
-if not backup_found:
-    logging.info("Skipped restore of missing backup {}".format(backup_name))
+if __name__ == '__main__':
+    if len(sys.argv) > 3:
+        config_file_path = sys.argv[2]
+        restore_key = sys.argv[3]
+    else:
+        logging.error("Usage: {} <config_file_path> <restore_key>".format(sys.argv[0]))
```

Comment:
Where is `sys.argv[1]` ? The helper says 2 parameters, the length check wants 3 (I assume argv[0] is the command).